### PR TITLE
enable LSU + Int8

### PR DESCRIPTION
**Pull request by nakajee**
- added code for LSU reduction for Int
- fixed incorrect vgpr index for PackData_INT8
- added test cases

### CODE REVIEW

**Comment by nakajee** [comment]
I could not reproduce CI test fail (lsu.yaml) on my local environment.
The log showed that the error was related to GSU post code for I8, but I did not enable GSU for I8 in lsu.yaml.
I moved LSU+I8 test cases in a new file (lsu_i8.yaml) and added GSU cases to see if it fails again or not.

File: tensilelite/Tensile/Tests/common/gemm/lsu_i8.yaml [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+          - [16, 16, 32, 1,  1,   4, 1,  2,1 ]
+          - [16, 16, 32, 1,  1,   1, 1,  1,1 ]
+          - [16, 16, 32, 1,  1,   2, 1,  1,1 ]
+          - [16, 16, 32, 1,  1,   4, 1,  1,1 ]
+          - [16, 16, 32, 1,  1,   1, 2,  1,1 ]
+          - [16, 16, 32, 1,  1,   1, 4,  1,1 ]
+          - [16, 16, 32, 1,  1,   2, 2,  1,1 ]
+        - WorkGroup:
+          - [16,4,1]
+          #- [8,4,2]
```
Comment (hcman2) [inline tensilelite/Tensile/Tests/common/gemm/lsu_i8.yaml]: LSU disabled?

Comment (nakajee) [reply tensilelite/Tensile/Tests/common/gemm/lsu_i8.yaml]: This was an experimental change.
I was suspecting CI error was not related to my change.
I will make some more experimental change in this PR and try I8 test without my change (to see if the error is related to my change or not).
So far, I am unable to reproduce the CI error in my local.

**Comment by hcman2** [comment]
https://ontrack-internal.amd.com/browse/SWDEV-454948
Looks like there exist a compiler bug for I8II cases. 
The I8I8I is OK.
You may separate the I8II test yaml and set it "xfail" first.
Compiler is targeting a fix in rocm6.2 rc4 version.

**Comment by nakajee** [comment]
I added back I8 LSU change and commented out I8II test case.
Hopefully, everything is OK with this change.

**Comment by nakajee** [comment]
2 tests failed.
[31mFAILED[0m Tensile/Tests/common/test_config.py::[1mtest_config[Tensile/Tests/common/groupedgemm/grouped_gemm_userargs.yaml][0m - SystemExit: 1
[31mFAILED[0m Tensile/Tests/common/test_config.py::[1mtest_config[Tensile/Tests/common/sparse/spmm_i8_sb.yaml][0m - SystemExit: 1

I could not reproduce the fail on my local.
I checked the detail of the fail.
Result of device is 0.
I saw similar random fails in Tensile CI before.
I suspect some memory copy issue, but not completely solved in Tensile yet.
It will not occur again when we re-run the CI test.
I will re-run CI test.

**Comment by nakajee** [comment]
OK. Finally gfx942 precheckin test passed including lsu_i8.
(ignoring known issue with rhel9)

**Review by hcman2 — APPROVED** [review]

**Review by AlexBrownAMD — APPROVED** [review]

**Pull request merged**